prinz/src/main/scala/pl/touk/nussknacker/prinz/mlflow/model/RegisteredModel.scala

```diff
@@ -11,14 +19,52 @@ case class RegisteredModelTag(key: String, value: String)
 case class RegisteredModelVersion(name: String, version: String, creation_timestamp: String, last_updated_timestamp: String,
                                   current_stage: String, source: String,
                                   run_id: String, status: String, tags: List[ModelVersionTag]) extends ModelVersion {
-  override def getModelInstance(): MLFModelInstance = new MLFModelInstance(new URL("http://localhost:1234"))
+
+  override def getModelInstance: Option[MLFModelInstance] =
+    downloadSignature(EXPERIMENT_ID, run_id)
+      .map(new MLFModelInstance(new URL("http://0.0.0.0:1234"), _))
```

Comment:
URL should not be hardcoded. There's `MODEL_PORT` in docker-compose settings, I think we could pass it down from env to here.

Reply:
will be done when env configured with variables in #40